addon/-private/system/model/record-data-map.ts

```diff
@@ -0,0 +1,53 @@
+import { DEBUG } from '@glimmer/env';
+
+export const RECORD_DATA_KEY = `${Date.now()}-record-data`;
+
+// TODO is there a way to import classes without causing cyclical imports for rollup?
+//   maybe strip "typing" imports prior to rollup?
```

Comment:
I'm not quite sure what the TODO here is?

Reply:
This is for @mike-north, basically in order to use classes as "types" we import them, but rollup doesn't like cyclical imports and sometimes this causes cyclical imports.